packages/webapp/src/components/FilterDropdown/Markup.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { MenuItem, CssBaseline } from '@material-ui/core';
```

Comment:
Why are you importing `CssBaseline`?

Reply:
I tried something and forgot to remove it at a later stage. Removed.